config.json

```diff
@@ -1056,6 +1056,20 @@
       "uuid": "0c17444e-65b0-4bc8-a7e1-ee6ab8c2e078"
     },
     {
+      "core": false,
+      "difficulty": 8,
+      "slug": "book-store",
+      "topics": [
+        "integers",
+        "lists",
+        "algorithm",
```

Comment:
algorithm -> algorithms

There is a [handy reference of the topics](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt). We try to stick with those and it helps to keep small differences (like plural/non-plural)from cropping up. Otherwise things could get messy! :smile: